### PR TITLE
Updated init to support Quick Start setup

### PR DESCRIPTION
**Pull request by revanth0212**
## Description

In an effort to make it easy for mesh developers to get started we have embarked on a quick start template to setup and test meshes both on the local and on github codespaces.

This PR adds the same template files to the init command to make the output of init command to look similar to the output of the template.

Draft template: https://github.com/revanth0212/api-mesh-starter-kit

## Related Issue

https://jira.corp.adobe.com/browse/CEXT-3347

## Motivation and Context

Better DevEx

## How Has This Been Tested?

This structure has already been validated on the quick start template.

Steps:

1. Run init command
2. Update mesh.json/.env with the necessary details
3. Run start command `yarn start mesh.json`
4. Alternatively run `aio api-mesh run mesh.json`
5. Run debug command `yarn debug mesh.json`
6. Alternatively run `aio api-mesh run mesh.json --debug`
7. Upload the init folder to github and create a codespace
8. Run start and debug command to verify the same behavior (aio api-mesh run does not work directly on codespaces)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/templates/readme.md
Comment (revanth0212) [inline src/templates/readme.md:1]: This is the sample readme from the quick start guide. We need to change this for the init command output. @jhadobe I could use your help to modify this.

Comment (jhadobe) [reply src/templates/readme.md:1]: @revanth0212 please review this PR https://github.com/AdobeDocs/graphql-mesh-gateway/pull/216. The readme in this repo should link to the documentation site after it is published.

Comment (revanth0212) [reply src/templates/readme.md:1]: Do you have a potential link for the published doc page? Ill add it the readme and the link will be active once the docs are published. Anyways these 2 PRs will go live at the same time.

Also, what do you think the contexts of this readme should be? Just a heading and a link to the doc page or something else?

Comment (jhadobe) [reply src/templates/readme.md:1]: @revanth0212 it depends when you want this page to publish. i'm waiting on some url forwarding. the destination will be: https://developer.adobe.com/graphql-mesh-gateway/mesh/template (after the merge), which should take place in the next day or so (just waiting on the release to be published).

I think it should contain a brief overview of the files and their purpose, but yes, mainly a link to the docs.

**Review by brasewel — DISMISSED** [review]

**Review by kmaschi — DISMISSED** [review]

**Review by jward-magento — DISMISSED** [review]

**Review by kmaschi — APPROVED** [review]

**Review by jhadobe — APPROVED** [review]

**Review by dthampy — APPROVED** [review]

**Pull request merged**
